Source/WebCore/Modules/model-element/DDModelPlayer.mm

```diff
@@ -141,75 +141,78 @@ void setOpaque(bool opaque)
 {
     RefPtr corePage = m_page.get();
     m_modelLoader = nil;
-    if (!m_modelLoader) {
```

Comment:
this is mostly removed indentation because the conditional was silly, `m_modelLoader` is set to nil on the line above